shared/AppInsightsCommon/src/ConnectionStringParser.ts

```diff
@@ -37,6 +37,10 @@ export function parseConnectionString(connectionString?: string): ConnectionStri
 
         // apply the default endpoints
         result.ingestionendpoint = result.ingestionendpoint || DEFAULT_BREEZE_ENDPOINT;
+        
+        if (strEndsWith(result.ingestionendpoint, "/")) {
+            result.ingestionendpoint = result.ingestionendpoint.slice(0,-1);
```

Comment:
We probably should also do this for the snippet as well, although we won't be able to use `strEndsWith` in the snippet or the string `endsWith` as it's not supported on IE and the snippet needs to work for IE :-(.

nit: I don't normally use `slice` on strings, but it works for this case and it's available on all browsers.